packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -2,18 +2,25 @@
   "running_tasks_title_text": "Running Tasks",
 
   "force_auth_web_login_authorize_dev_hub_text": "SFDX: Authorize a Dev Hub",
-  "force_org_create_default_scratch_org_text": "SFDX: Create a Default Scratch Org...",
+  "force_org_create_default_scratch_org_text":
```

Comment:
@ruthemmanuelle - Have these messages been approved? They come from the CLI in Palette spreadsheet but I'm not sure if these have been approved yet.

Reply:
Yes, the messages in my column of the spreadsheet are fine to use. I haven't been running VS Code messages by the UI Text Reviews team, since their primary focus is on UI text and error messages for the core app. Just having me review these messages is fine.